files/deployment.yaml

```diff
@@ -29,52 +29,71 @@
       command: oc whoami -t
       register: kubeconfig_token
       become: "{{ as_root }}"
+
+    # path_to_secrets = this is what we get from the outside
+    # internal_secrets_path = this is what we'll use here
+    - set_fact: secrets_path={{ deployment_dir }}/secrets/dev
```

Comment:
Can you please send the deployment changes in another PR? (This started to be really unreviewable.)

Reply:
yes

Reply:
Thanks!